src/sqlfluff/rules/L070.py

```diff
@@ -0,0 +1,249 @@
+"""Implementation of Rule L070."""
+from typing import Optional
+
+from sqlfluff.utils.analysis.select_crawler import Query, SelectCrawler, WildcardInfo
+from sqlfluff.core.rules import BaseRule, LintResult, RuleContext
+from sqlfluff.core.rules.crawlers import SegmentSeekerCrawler
+from sqlfluff.core.rules.doc_decorators import document_groups
+
+
+@document_groups
+class Rule_L070(BaseRule):
+    """Queries within set query produce different numbers of columns.
+
+    **Anti-pattern**
+
+    When writing set expressions
+    , all queries must return the
+    same number of columns
```

Comment:
Is there a reason for the lines being so short? Also, the sentence needs a `.` at the end.